### ISSUE TITLE
No module named `pyastgrep.api`: using pyastgrep as library

### ISSUE THREAD

**Issue by nduckwiler** [spookylukey/pyastgrep#20]
I can only access the Python library using `import pyastgrep.search` and `pyastgrep.search.search_python_files()`. Am I doing something wrong, or does [the documentation](https://pyastgrep.readthedocs.io/en/latest/library.html) need an update?

I tried importing `pyastgrep` and using `pyastgrep.api.search_python_files` and got this error:
```
AttributeError: module 'pyastgrep' has no attribute 'api'
```

I also tried importing `pyastgrep.api` and got this error:
```
ModuleNotFoundError: No module named 'pyastgrep.api'
```

This is running Python 3.11.6 on a Mac M2 with MacOS Ventura 13.5.2. I installed `pyastgrep` with `pip3 install pyastgrep`.

```
pyastgrep --version
pyastgrep 1.3.1, Python 3.11.6 (main, Nov  2 2023, 04:39:43) [Clang 14.0.3 (clang-1403.0.22.14.1)]

which pyastgrep
/opt/homebrew/bin/pyastgrep

pip3 --version
pip 23.3.1 from /opt/homebrew/lib/python3.11/site-packages/pip (python 3.11)
```

**Comment by spookylukey** [comment]
Thanks for the report. It looks like I have not released those changes, nor have I made that clear in docs. I will fix soon!

**Comment by spookylukey** [comment]
Fixed now in released 1.3.2
